spec/spec-suite-jasmine.js

```diff
@@ -0,0 +1,8 @@
+'use babel'
+/* @flow */
+
+require('./spec-suite')
+
+module.exports = function () {
```

Comment:
Basically, I wasn't sure what your conventions are. It would be good to get set up with ESLint.